src/app/shell/Destiny.tsx

```diff
@@ -28,7 +28,7 @@ class Destiny extends React.Component<Props> {
     // Define some hotkeys without implementation, so they show up in the help
     const hotkeys: Hotkey[] = [
       {
-        combo: 'i',
+        combo: 't',
```

Comment:
wonder if this should just be deprecated, as it will change again soon when the popup is redesigned?

Reply:
Yeah, it's awkward. I haven't made a ton of progress on the new item popup recently.